tests/ci/submit_azureml_pytest.py

```diff
@@ -165,7 +181,15 @@ def create_run_config(cpu_cluster, docker_proc_type, conda_env_file):
     run_amlcompute = RunConfiguration()
     run_amlcompute.target = cpu_cluster
     run_amlcompute.environment.docker.enabled = True
-    run_amlcompute.environment.docker.base_image = docker_proc_type
+
+    # Allow use of a custom Docker image
+    if use_custom_docker:
+        run_amlcompute.environment.docker.base_image = custom_docker_image
+        run_amlcompute.environment.docker.base_image_registry.address = custom_docker_image_repo
+        run_amlcompute.environment.docker.base_image_registry.username = custom_docker_image_repo_username
```

Comment:
not sure it's secure to pass username+pw as it may end up in log files?

Reply:
Yea I am not sure about this, either. To make it secure probably we need to follow the approach with Azure Keyvault.